distributedlog-core/pom.xml

```diff
@@ -198,6 +198,34 @@
             </configuration>
           </execution>
           <execution>
+            <id>shade-bk</id>
+            <phase>package</phase>
+            <goals>
+              <goal>shade</goal>
+            </goals>
+            <configuration>
+              <createDependencyReducedPom>true</createDependencyReducedPom>
+              <shadedArtifactAttached>true</shadedArtifactAttached>
+              <shadedClassifierName>shaded-bk</shadedClassifierName>
```

Comment:
should we use 'bkshade' instead of 'shaded-bk'?

Reply:
ok, I have updated.